src/main/java/racingcar/car/NextStepCars.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.car;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class NextStepCars {
+    private List<Car> nextStepCars;
+
+    public NextStepCars(List<Car> nextStepCars) {
+        this.nextStepCars = nextStepCars;
+    }
+
+    public NextStepCars(String carList) {
+        nextStepCars = new ArrayList<>();
+        for(String car : carList.split(",")) {
+            nextStepCars.add(new NextStepCar(car));
+        }
+    }
+
+    public List<Car> getNextStepCars() {
+        return nextStepCars;
+    }
+
+    public int carCount() {
+        return nextStepCars.size();
+    }
+
+    public List<String> getWinner() {
+        int winnerPosition = 0;
+        for(Car car : this.nextStepCars) {
+            winnerPosition = Math.max(winnerPosition, car.getPosition());
+        }
+        int finalWinnerPosition = winnerPosition;
+        List<String> collect = this.nextStepCars.stream()
+                .filter(car -> car.getPosition() == finalWinnerPosition)
```

Comment:
아래와 같이 자동차 객체에게 `우승자니?` 물어보는 방향으로 하면 어떤 장점이 있을까요 :) ?
해당 피드백과 같이 고민해보시면 좋을 것 같아요 😄 
```suggestion
                .filter(car -> car.isWinner(finalWinnerPosition))
```